### PR TITLE
[zh-cn] sync tasks/run-application/*

### PR DESCRIPTION
**Pull request by my-git9**
```
# Changes to be committed:
#       modified:   content/zh-cn/docs/tasks/run-application/configure-pdb.md
#       modified:   content/zh-cn/docs/tasks/run-application/delete-stateful-set.md
#       modified:   content/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod.md
#       modified:   content/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
#       modified:   content/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale.md
#       modified:   content/zh-cn/docs/tasks/run-application/run-replicated-stateful-application.md
#       modified:   content/zh-cn/docs/tasks/run-application/run-single-instance-stateful-application.md
#       modified:   content/zh-cn/docs/tasks/run-application/run-stateless-application-deployment.md
#       modified:   content/zh-cn/docs/tasks/run-application/scale-stateful-set.md

```

### CODE REVIEW

File: content/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 1]
```diff
@@ -411,8 +415,8 @@ or _flapping_. It's similar to the concept of _hysteresis_ in cybernetics.
 ## 工作量规模的稳定性 {#flapping}
 
 在使用 HorizontalPodAutoscaler 管理一组副本的规模时，由于评估的指标的动态特性，
-副本的数量可能会经常波动。这有时被称为 **抖动（thrashing）** 或 **波动（flapping）**。
-它类似于控制论中的 **滞后（hysteresis）** 概念。
+副本的数量可能会经常波动。这有时被称为**抖动（thrashing）**或**波动（flapping）**。
+它类似于控制论中的**滞后（hysteresis）**概念。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale.md:419]: This doesn't work as expected. Please check the preview.
Removing this extra spaces may confuse the buggy interpreter.

Comment (my-git9) [reply content/zh-cn/docs/tasks/run-application/horizontal-pod-autoscale.md]: fixed

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**